src/sentry/grouping/ingest/seer.py

```diff
@@ -147,6 +154,32 @@ def _ratelimiting_enabled(event: Event, project: Project) -> bool:
     return False
 
 
+def _circuit_breaker_broken(event: Event, project: Project) -> bool:
+    circuit_broken = not seer_similarity_circuit_breaker.should_allow_request()
```

Comment:
For some reason, GitHub was not helping me get to the source code for this.

Here's a link in case it happens to anyone else after me:
https://github.com/getsentry/sentry/blob/e88dfb903ad1428c5d44e2b20aef172562222c0d/src/sentry/utils/circuit_breaker2.py#L269-L281